lib/VolSurface/Calibration/Equities.pm

```diff
@@ -0,0 +1,637 @@
+package VolSurface::Calibration::Equities;
```

Comment:
Torsten, everytime we get a new vol surface this is used- so maybe every 30mins. As of now we are only intending to move this to its repo, maybe in future we can try to convert to C or other language, but not as of now.